localgov_publications.info.yml

```diff
@@ -2,10 +2,10 @@ name: LocalGov Publications
 type: module
 description: HTML publications for the LocalGovDrupal distribution.
 package: LocalGov Drupal
-core_version_requirement: ^9 || ^10
+core_version_requirement: ">=9"
```

Comment:
I think this is too loose and caused issues in the past where `>=9` was used and the module didn't yet work on D11. We no longer support Drupal 9 so this be changed to the actual supported versions e.g. `^10 || ^11` similar to the next alert banner version https://github.com/localgovdrupal/localgov_alert_banner/blob/1.8.x/localgov_alert_banner.info.yml

Reply:
Is it a case of making our lives easier for testing modules if we are relaxed about it, but possibly making life difficult for others if the module doesn't actually live up to upgradablity? But maybe we should just list the versions we test against, and expand them when testing is expanded?

Reply:
It is probably more a safety net as we are being specific at the version level of what Drupal Core we support and not letting the user assume we do support a specific version. If I remember correctly for drupal.org modules they normally recommend to specify the core versions supported.

For example if a module was using some functionality that is no longer in a specific core version there is a case where it could whitescreen the website.

However if we do loosen the version requirements we could have a page on the docs website which states Drupal core support for both the projects and each module

Reply:
This isn't a hill I want to die on, but if someone wants to run the module on D12 pre release, I don't see why we should stop them.

Also if we're taking away D9 support I think technically we should make a new major release, as it could be a breaking change for some people. (Although if we know all the councils out there running publications are on D10+ then we could just go for it.)

Reply:
I've made the same change as the other LGD modules now.